app/controllers/application_controller.rb

```diff
@@ -3,13 +3,19 @@ class ApplicationController < ActionController::Base
 
   rescue_from Pundit::NotAuthorizedError, with: :render_not_authorized
 
-  before_action :check_maintenance
+  before_action :check_maintenance_status
   before_action :set_paper_trail_whodunnit
 
-  def check_maintenance
-    if FeatureToggle.maintenance_mode_enabled? && !%w[service-unavailable accessibility-statement privacy-notice cookies].include?(request.fullpath.split("?")[0].delete("/"))
-      redirect_to service_unavailable_path
-    elsif !FeatureToggle.maintenance_mode_enabled? && request.fullpath.split("?")[0].delete("/") == "service-unavailable"
+  def check_maintenance_status
+    if FeatureToggle.service_moved?
```

Comment:
I've been wondering about some way to make this bit a little nicer, just to avoid having so many conditions, but I don't think it'd be worth the effort.

Reply:
I agree 👍